data.sql

```diff
@@ -13,4 +13,179 @@ VALUES
     ('Boarmon', '2005-06-07', 7, true, 20.4),
     ('Blossom', '1998-10-13', 3, true, 17),
     ('Ditto', '2022-05-14', 4, true, 22);
+
+/*** Insert the following data into the owners table: ***/
+
+/* 
+Sam Smith 34 years old. 
+Jennifer Orwell 19 years old.
+Bob 45 years old.
+Melody Pond 77 years old.
+Dean Winchester 14 years old.
+Jodie Whittaker 38 years old.
+
+*/ 
+
+INSERT INTO owners (full_name, age)
+VALUES
+    ('Sam Smith', 34),
+    ('Jennifer Orwell', 19),
+    ('Bob', 45),
+    ('Melody Pond', 77),
+    ('Dean Winchester', 14),
+    ('Jodie Whittaker', 38);
+
+/*
+    Insert the following data into the species table:
+    Pokemon
+    Digimon
+*/
+
+INSERT INTO species (name)
+VALUES 
+    ('Pokemon'),
+    ('Digimon');
+
+
+/***   Modify your inserted animals so it includes the species_id value:  ***/
+
+ /*
+  -If the name ends in "mon" it will be Digimon
+  -All other animals are Pokemon 
+*/
+
+BEGIN TRANSACTION;
+UPDATE animals 
+SET species_id = 2
+WHERE name LIKE '%mon';
```

Comment:
- [ ] Please ensure that everything that populates tables with data should is only saved in the data.sql file because I can see that you added query data here and also data that create or change the structure of the table which should be saved in the schema.sql file. Kindly format the data.sql file properly.

Reply:
@vickymarz , Thank you for your feedback, but those referred lines are only to change DATA, maybe you are talking about these lines: 
![for_FEEDBACK](https://user-images.githubusercontent.com/26579035/228989871-5f81f779-0a58-49ec-b85a-62c43a3cb939.png)


I am using a query there (line 84) to retrieve first the id of the owner and then I use it to update the owner of the specific animals, I am not changing any structure of the table. I am in the case that I don't know the id of the owner and only the name. So that is my approach to update the DATA.